src/main/resources/liquibase/changeset/07.01.2022_1200_insert_tags_data.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
+    <changeSet id="07.01.2022_1200_insert_tags_data" author="aliaksandr">
+        <insert tableName="tags">
+            <column name="id" value="1"/>
```

Comment:
Interesting
You use sequence(or autoincrement) and at the same time you set id 

Please learn theory about database id and fix this

Reply:
Ok